site/content/en/docs/start/_index.md

```diff
@@ -534,7 +534,7 @@ minikube dashboard
 
 <h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">4</strong></span>Deploy applications</h2>
 
-Create a sample deployment and expose it on port 8080:
+Create a sample deployment and expose it on port 8080, if you are using Apple M1, replace the image with `k8s.gcr.io/echoserver-arm:1.8`:
```

Comment:
note that this is an armv7 (arm32) image, and not a arm64 image... It needs to be updated, to a newer nginx, for that.

Reply:
```
REPOSITORY                  TAG       IMAGE ID       CREATED       SIZE
k8s.gcr.io/echoserver       1.4       a90209bb39e3   6 years ago   140MB
k8s.gcr.io/echoserver-arm   1.8       72565bf5bbed   5 years ago   85MB
```

Reply:
* https://github.com/kubernetes/minikube/issues/11107